std/typecons.d

```diff
@@ -3793,8 +3793,28 @@ Params:
                 sink.formatValue(_value, fmt);
             }
         }
+
+        void toString()(scope void delegate(const(char)[]) sink, scope const ref FormatSpec!char fmt) const
```

Comment:
does making the toString above this one `inout` instead of `const` fix the issue? 
if so please do that.

Reply:
That would fix the issue but it would add the `inout` qualifier in certain cases:
```d
struct Test { string s; }
Nullable!(Test, Test("null")) nt = Test("test");
writeln(nt);  // inout(Test)("test")
```
Do we want this?

Reply:
Nope